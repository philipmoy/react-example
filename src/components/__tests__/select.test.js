import React from 'react';
import { shallow } from 'enzyme';

import CustomSelect from '../select';

describe('CustomSelect', () => {
  it('renders correctly', () => {
    const wrapper = shallow(
      <CustomSelect
        options={[]}
      />,
    );
    expect(wrapper).toMatchSnapshot();
  });
});
