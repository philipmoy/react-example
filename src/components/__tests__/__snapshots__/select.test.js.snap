// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CustomSelect renders correctly 1`] = `
<div
  className="custom-select"
/>
`;
